contrib/tlp/files/tlp

```diff
@@ -0,0 +1,7 @@
+# tlp service
+
+type            = scripted
+depends-on      = local.target
+waits-for       = network.target
```

Comment:
as per the discussion in this PR, this should have `after = networkmanager` in it

Reply:
.... no?

@Natthias
> ... tlp-rdw is the only part of tlp which depends on NetworkManager, and is only a small subset of what tlp provides.

and tlp-rdw has been split to a separate package.

---

I did comb through the code just to look if `tlp init start` invokes anything from `tlp-rdw` ... and it doesn't seem like it does...


IIRC it seemed to me that all the tlp-rdw stuff was invoked by the  
`usr/lib/NetworkManager/dispatcher.d/99tlp-rdw-nm`

...idk :shrug:

Reply:
@Natthias ?

Reply:
Yea, tlp-rdw doesn't seem to actually need to have tlp running, so leaving out the `after = networkmanager` seems fine. I also tried testing it briefly and tlp-rdw seemed to work fine without tlp actually running, all it needed was networkmanager.

Bit odd that upstream has it start after networkmanager, so maybe there is a reason, but not one I can see either.